lib/helper/WebDriverIO.js

```diff
@@ -1702,4 +1712,18 @@ function isFrameLocator(locator) {
   return locator[key];
 }
 
+function prepareLocateFn(context) {
+  if (!context) return this._locate.bind(this);
+  let el;
```

Comment:
Don't understand, where we define `el`? Could you please give some comments about it?